lucene/suggest/src/java/org/apache/lucene/search/suggest/document/NRTSuggesterBuilder.java

```diff
@@ -43,6 +44,9 @@
    * in the output
    */
   public static final int PAYLOAD_SEP = '\u001F';
```

Comment:
You can set it to TokenStreamToAutomaton.POS_SEP directly and add a test to ensure that PAYLOAD_SEP and SEP_LABEL are equals to `0x001F` for BWC.